src/web/components/SmallHeadline.tsx

```diff
@@ -96,6 +103,7 @@ export const SmallHeadline = ({
     size = 'xxsmall',
     coloured = false,
     linkTo,
+    visitedColour = '',
```

Comment:
Maybe the default could be just `undefined`

Reply:
Cool. Updated.